### PR TITLE
[NG][Formly] Radio & Checkboxes

### PR DESCRIPTION
**Pull request by sanlucca**
Adds radio & checkboxes for Formly.
Radio aspects is changing based on the number of options, using UX guidelines.
Merge #386 before

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://387-pr-lucca-front-LuccaSA.surge.sh

File: packages/ng/libraries/formly/src/lib/formly.config.ts [hunk 1]
```diff
@@ -43,6 +47,16 @@ export const LU_FORMLY_CONFIG = {
...
+		{
+			name: 'radio',
+			component: LuFormlyFieldRadios,
+			wrappers: ['textfield-layout'],
+		},
+		{
+			name: 'checkbox',
+			component: LuFormlyFieldCheckboxes,
+			wrappers: ['textfield-layout'],
+		},
```
Comment (lucienbertin) [inline packages/ng/libraries/formly/src/lib/formly.config.ts]: i thought we introduced the notion of `textfield` insted of `field` cuz _standard_ inputs (text, select, textarea) dont behave like checkboxes and radios

File: packages/scss/src/components/inputs/textfield/_textfield.scss [hunk 2]
```diff
@@ -225,6 +225,21 @@
 	}
 }
 
+/* Style can & should be applied at the form level */
+.form {
+	&.mod-material .textfield {
+		@extend %input_material;
```
Comment (lucienbertin) [inline packages/scss/src/components/inputs/textfield/_textfield.scss]: <3

File: packages/ng/libraries/formly/src/lib/formly.config.ts (hunk 1 above)
Comment (sanlucca) [reply packages/ng/libraries/formly/src/lib/formly.config.ts]: @jeremie-lucca : Lucien is raising a good point. Should I rename the whole `textfield` into `form-field` ?

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://387-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://387-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://387-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/inputs/checkbox/_checkboxesfield.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+%checkboxesfield-row {
+	display: flex;
+	flex-wrap: wrap;
+	.radio {
+		margin-top: 0;
+		margin-bottom: 0;
+	}
+	.radio {
```
Comment (jeremie-lucca) [inline packages/scss/src/components/inputs/checkbox/_checkboxesfield.scss]: Double .radio declaration

File: packages/scss/src/components/inputs/radio/_radiosfield.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+	.radiosfield-input {
+		@extend %radiosfield-row;
+	}
+}
+
+%radiosfield-row {
+	display: flex;
+	flex-wrap: wrap;
+	.radio {
```
Comment (jeremie-lucca) [inline packages/scss/src/components/inputs/radio/_radiosfield.scss]: Double .radio declaration, bis.

**Pull request merged**